test/cron/monthly/host-settlement.test.js

```diff
@@ -255,10 +255,6 @@ describe('cron/monthly/host-settlement', () => {
     expect(gphHostSettlementExpense).to.have.nested.property('data.isPlatformTipSettlement', true);
   });
 
-  it('should use a payout method compatible with the host currency', () => {
-    expect(gphHostSettlementExpense).to.have.property('PayoutMethodId', 2956);
-  });
-
```

Comment:
Rather than removing it, can we update the test to ensure it picks an appropriate payout method?